templates/add_group.html

```diff
@@ -101,6 +103,22 @@ <h1>Family [[ new_group.name ]]</h1>
                     >
                     <label class="custom-control-label" for="male">Male</label>
                 </div>
+                <div class="mt-3">
+                    <div class="form-check">
+                        <input class="form-check-input" type="checkbox" v-model="refugees[i].disabilities"
+                               id="check_disabilities" required>
+                        <label class="form-check-label" for="check_disabilities">
+                            Disabilities
+                        </label>
```

Comment:
i dont see field "disabilities" in refugee model , but there is field "need_barrier_free", is this the same thing? In the mockup the checkbox label says disabilities @Lassulus

Reply:
ah yes, it was renamed in a meeting

Reply:
great, fixed 👍🏻